tinygrad/tensor.py

```diff
@@ -162,8 +162,14 @@ def scaled_uniform(*shape, **kwargs) -> Tensor: return Tensor.uniform(*shape, **
   @staticmethod
   def glorot_uniform(*shape, **kwargs) -> Tensor: return Tensor.uniform(*shape, **kwargs).mul((6/(shape[0]+prod(shape[1:])))**0.5)
 
-  # ***** toposort and backward pass *****
+  # ***** kaiming uniform to weights *****
+  # https://pytorch.org/docs/stable/_modules/torch/nn/init.html#kaiming_uniform_
+  @staticmethod
+  def kaiming_uniform(*shape, a:float = 0.01, **kwargs) -> Tensor:
+    bound = math.sqrt(3.0) * math.sqrt(2.0 / (1 + a ** 2)) / math.sqrt(shape[1] * prod(shape[2:]))
+    return Tensor(LazyNumpyArray(lambda lna: np.random.uniform(low=-bound, high=bound, size=lna.shape), shape, np.float32))
```

Comment:
Use `Tensor.uniform` like the above init functions.

Reply:
But `Tensor.uniform` does not have bound parameters for low and high bounds. What should I do?

Reply:
Hey, @geohot, I updated `Tensor.uniform` since it wasn't generalizable to a specific range, according to [1]. So, I was able to change `return` line. It is an unpalatable practice to update two functions, although it works fine, in a pull request, but I had to do it.

[1] https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.uniform.html#numpy.random.Generator.uniform